os/kernel/debug/sysdbg.c

```diff
@@ -416,60 +326,86 @@ static void sysdbg_print(void)
 
 static ssize_t sysdbg_write(FAR struct file *filep, FAR const char *buffer, size_t buflen)
 {
-	char arg1[32] = { '\0' };
-	char arg2[32] = { '\0' };
+	char option[32] = { '\0' };
+	char count[32] = { '\0' };
```

Comment:
What is the reason for using 32 chars for option and count?

Reply:
Thankyou for pointing it out.
Though it was used from old code. I checked the sysdbgapp passes a buffer with length 64. so it might have 32 chars of arg1 and arg2. 

I think since we have both the parameters to be used as numerics the length can be decreased. Please let me know your suggestion.